nlu.py

```diff
@@ -4,21 +4,8 @@
 command_map = {'曠課': '昌昌;曠課', '課表': '昌昌;課表', '成績': '昌昌;成績'}
 
 
-def response_message(text: str) -> list:
-    messages = []
-    cmds = _get_commands(jieba.cut(text))
-    for cmd in cmds:
-        messages.append(_exec_command(cmd))
-    return messages
-
-
-def _get_commands(text_sequence: [str]) -> set:
-    possible_commands = set()
-    for text in text_sequence:
-        cmd = command_map.get(text)
-        if cmd is not None:
-            possible_commands.add(cmd)
-    return possible_commands
+def similarity(a: str, b: str) -> float:
+    return _similarity(jieba.lcut(a), jieba.lcut(b))
```

Comment:
Why lcut not cut?

Reply:
according to [here](generator).

`jieba.cut` return an iterable generator.
`jieba.lcut` return an list.

Reply:
And would that cause any problem?
I don't need to know the details of the function; I want to know what's different between the solution

Reply:
code below
```python
def _similarity(a: [str], b: [str]) -> float:
    words = set(a + b)
    vector_a = _get_vec(words, a)
    vector_b = _get_vec(words, b)
    return _consine_similarity(vector_a, vector_b)
```
we need to get all-words by HashSet from jieba segmentation and The HashSet only accept the type of list and not a generator.
It's why I choose `lcut` and not `cut`.